lib/kitchen/provisioner/dokken.rb

```diff
@@ -31,6 +31,10 @@ class Dokken < Kitchen::Provisioner::ChefZero
       plugin_version Kitchen::VERSION
 
       default_config :root_path, '/opt/kitchen'
+      default_config :chef_binary, '/opt/chef/embedded/bin/chef-client'
```

Comment:
Was thinking of leaving it like this, this way the "old" way does not change but peoiple can change it in their configs instead, makes more sense to me. Dont fix what is not broken....